CHANGELOG.md

```diff
@@ -40,6 +40,8 @@ _ `_optional` subpackage for managing optional dependencies
 
 ### Fixed
```

Comment:
@AdrianSosic we might still have to do some warnings catching or call adjustments in the simulation package. It emits warnings for example scripts where the `RandomRecommender` is called in a simulation. 

I think in such a simulation its very much intended that a user uses a non-predictive recommender in all iterations. Instead I think it would be good if the simulator would decide whether to pass `objective` and `measurements` based in the recommender class (similar to the logic in this PR).

Reply:
Sounds reasonable. Do you see it part of this PR or of the next upcoming simulation refactoring PRs?

Reply:
see it after/during other parts of the simulation refactoring

Reply:
ok, then I resolve